File: stylebook/stylebook/colors.py
from colorama import Fore, Style


def cyan(content: str) -> str:
    return f'{Fore.CYAN}{content}{Style.RESET_ALL}'


def blue(content: str) -> str:
    return f'{Fore.BLUE}{content}{Style.RESET_ALL}'


def green(content: str) -> str:
    return f'{Fore.GREEN}{content}{Style.RESET_ALL}'


def red(content: str) -> str:
    return f'{Fore.RED}{content}{Style.RESET_ALL}'


def yellow(content: str) -> str:
    return f'{Fore.YELLOW}{content}{Style.RESET_ALL}'


def d(content: str) -> str:
    return f'{Style.DIM}{content}{Style.RESET_ALL}'


def b(content: str) -> str:
    return f'{Style.BRIGHT}{content}{Style.RESET_ALL}'


def i(content: str) -> str:
    return f'\033[3m{content}{Style.RESET_ALL}'

File: cmd/colors.go
package cmd

import "github.com/fatih/color"

func Cyan(text string) string {
	return color.New(color.FgCyan).Sprint(text)
}

func Blue(text string) string {
	return color.New(color.FgBlue).Sprint(text)
}

func Green(text string) string {
	return color.New(color.FgGreen).Sprint(text)
}

func Yellow(text string) string {
	return color.New(color.FgYellow).Sprint(text)
}

func Red(text string) string {
	return color.New(color.FgRed).Sprint(text)
}

func B(text string) string {
	return color.New(color.Bold).Sprint(text)
}

func D(text string) string {
	return color.New(color.Faint).Sprint(text)
}

func I(text string) string {
	return color.New(color.Italic).Sprint(text)
}
